extensions/modules/widget-utils/notes/feature-hit_test.md

```diff
@@ -0,0 +1,4 @@
+# Path hit testing
+
+* A function `hitTestPath` has been added providing hit testing of a
```

Comment:
Yes, that should work.

I wish there was a solution for this that didn't involve rendering the path to a bitmap... it's a well-defined geometrical operation, dammit!